AudioBar/AudioBarTests.swift

```diff
@@ -355,6 +374,69 @@ class AudioBarTests: XCTestCase, Tests {
         expect(view?.playbackTime, "-0:40")
     }
 
+    func testPlaybackDuration1() {
+        let view = expectView(for: .readyToPlay(.init(duration: 1)))
+        expect(view?.playbackDuration, 1)
+    }
+
+    func testPlaybackDuration2() {
+        let view = expectView(for: .readyToPlay(.init(duration: 20)))
+        expect(view?.playbackDuration, 20)
+    }
+
+    func testPlaybackDuration3() {
+        let view = expectView(for: .readyToPlay(.init(duration: 21)))
+        expect(view?.playbackDuration, 21)
+    }
+
+    func testPlaybackDuration4() {
+        let view = expectView(for: .readyToPlay(.init(duration: 60)))
+        expect(view?.playbackDuration, 60)
+    }
+
+    func testElapsedPlaybackTime1() {
+        let view = expectView(for: .readyToPlay(.init(currentTime: nil)))
+        expect(view?.elapsedPlaybackTime, 0)
+    }
+
+    func testElapsedPlaybackTime2() {
+        let view = expectView(for: .readyToPlay(.init(currentTime: 0)))
+        expect(view?.elapsedPlaybackTime, 0)
+    }
+
+    func testElapsedPlaybackTime3() {
+        let view = expectView(for: .readyToPlay(.init(currentTime: 1)))
+        expect(view?.elapsedPlaybackTime, 1)
+    }
+
+    func testElapsedPlaybackTime4() {
+        let view = expectView(for: .readyToPlay(.init(currentTime: 20)))
+        expect(view?.elapsedPlaybackTime, 20)
+    }
+
+    func testSeekIntervalWhenReadyToPlay() {
+        let view = expectView(for: .readyToPlay(.init()))
+        expect(view?.seekInterval, 15)
+    }
+
+    func testPlayAndPauseCommandsEnabled1() {
+        let view = expectView(for: .readyToPlay(.init(isPlaying: true)))
+        expect(view?.isPlayCommandEnabled, false)
+        expect(view?.isPauseCommandEnabled, true)
+    }
+
+    func testPlayAndPauseCommandsEnabled2() {
+        let view = expectView(for: .readyToPlay(.init(isPlaying: false)))
```

Comment:
Both `isPlayCommandEnabled` and `isPauseCommandEnabled` are derived not only from `isPlaying` and `isPlayPauseButtonEnabled`. In turn, `isPlayPauseButtonEnabled` is derived from `remainingTime` and that one is derived from `currentTime` and `duration`.

In unit tests, we have the following default initializer:

```swift
extension AudioBar.State.ReadyToPlay {
    init(isPlaying: Bool = false, duration: TimeInterval = 60, currentTime: TimeInterval? = nil) { ... }
}
```

I should be able to change all three arguments  — `isPlaying`, `duration` and `currentTime` — without any tests failing. When I change `currentTime` to `60`, this test fails. This means, you rely on "random data" we use there.